src/features/gtlPlaylist/hooks/useGTLPlaylists.native.test.ts

```diff
@@ -0,0 +1,59 @@
+import { SearchResponse } from '@algolia/client-search'
+
+import { Offer } from 'shared/offer/types'
+import { renderHook, waitFor } from 'tests/utils'
+
+import * as useGTLPlaylistsLibrary from '../api/gtlPlaylistApi'
+
+import { useGTLPlaylists } from './useGTLPlaylists'
+
+jest.mock('features/home/helpers/useHomePosition', () => ({
+  useHomePosition: jest.fn().mockReturnValue({
+    position: {
+      latitude: 2,
+      longitude: 2,
+    },
+  }),
+}))
+
+jest.mock('features/profile/helpers/useIsUserUnderage', () => ({
+  useIsUserUnderage: jest.fn().mockReturnValue(false),
+}))
+
+jest.spyOn(useGTLPlaylistsLibrary, 'fetchGTLPlaylists').mockResolvedValue([
+  {
+    title: 'Test',
+    offers: {
+      hits: [],
+    } as unknown as SearchResponse<Offer>,
+    layout: 'one-item-medium',
+  },
+])
+
+describe('useGTLPlaylists', () => {
+  it('should return correct data based on fetch api return', async () => {
```

Comment:
Je pense que tu peux avoir 2 tests ici aussi :
- `should fetch GTL playlists`
- `should return playlists information`
Je ne pense pas que j'aurai mis `expect(result.current).toEqual([])` autant se focus sur ton résultat de hook avec `waitFor`